app/components/pages/Live.vue

```diff
@@ -137,57 +86,35 @@
 }
 
 .studio-controls-panel {
-  flex-basis: 50%;
+  flex-grow: 1;
   padding-left: 0;
   .padding-right(2);
 }
 
-.sizer-container {
-  width: 100%;
-  .radius();
-  background-color: @day-secondary;
-  display: flex;
-  justify-content: flex-start;
-  align-items: flex-start;
-  position: relative;
-  .border();
-  border-top: 0;
-  height: calc(~'100% - 29px');
-}
-
-.sizer-items {
-  bottom: 8px;
-  right: 8px;
+.live-page-resizer {
+  margin: 4px 0;
 }
 
 .live-preview-container {
-  width: 300px;
   flex: 0 0 auto;
-  position: relative;
-}
-
-.live-preview-container__content {
-  height: 100%;
 }
 
 .live-display-wrapper {
-  width: 100%;
-}
-
-.live-display {
-  width: 100%;
-  height: 100%;
+  max-width: 100%;
+  .radius();
```

Comment:
Nitpick: put mixins above other rules.

Reply:
Is this an actual thing people do?  Why?